Added en text to community channel (#130)
DaoCloud/DaoCloud-docs: docs/en/community/cloudtty.md

# cloudtty：A Kubernates Cloud Shell (Web Terminal) Operator

> cloudtty is an easy-to-use operator to run web terminal and cloud shell intended for a kubernetes-native environment. You can easily open a terminal on your own browser via cloudtty. The community is always open for any contributor and those who want to have a try.

Reviewer: ```suggestion
cloudtty is an easy-to-use operator to run a web terminal and cloud shell intended for a kubernetes-native environment. You can easily open a terminal on your own browser via cloudtty. The community is always open for any contributor and those who want to have a try.
```

Literally, cloudtty herein refers to a virtual console, shell, or terminal running on web and clouds. You can use it anywhere with an internet conneciton and it will be automatically connected to the cloud.

!!! info "What is TTY"

>     Early user terminals connected to computers were electromechanical teleprinters or teletypewriters (TeleTYpewriter, TTY), which might be the origin of TTY. Gradually, TTY has continued to be used as the name for a text-only console although now this text-only console is a virtual console not a physical console.

Reviewer: ```suggestion
    Early user terminals connected to computers were electromechanical teleprinters or teletypewriters (TeleTYpewriter, TTY), which might be the origin of TTY. Gradually, TTY has continued to be used as the name for a text-only console, although now it is a virtual console not a physical one.
```

## Why Do You Need cloudtty?

A project [ttyd](https://github.com/tsl0922/ttyd) provides some features to share terminals over the web.
But if you use Kubernetes, a more cloud-native enviroment is required to run the webtty via the `kubernetes` way (running as a pod, and generated by CRDs),
which is covered by cloudtty. You are welcome to try cloudtty :tada:.

## Applicable Scenarios

cloudtty is applicable to the following scenarios:

1. Many enterprises use a cloud platform to manage Kubernetes, but due to security reasons,
   you cannot use SSH to connect the node host to execute `kubectl` commands.
   In this case, you may require a cloud shell capability.

> 2. A running container on kubernetes can be "entered" (via `Kubectl exec`) on a browser web page.

Reviewer: ```suggestion
2. A running container on Kubernetes can be "entered" (via `Kubectl exec`) on a browser web page.
```
